### DISCUSSION TITLE
Is there a reason why Forge version 47.1.3 is recommended?

### DISCUSSION THREAD

**Discussion by ghost** [Sinytra/Connector#930] [Q&A]
Let's say that I decide to use the Sinytra Connector with Forge Version 4.2.21, the latest version of Forge for 1.20.1. 

Has there been any known bugs with the higher versions of Forge with the Sinytra Connector? Or it is simply that 471.3 is the only version of Forge the connector has been tested on and every other version is unknown in terms of usability?

**Comment by MOST0AFA** [comment]
idk

**Comment by ooogerman** [comment]
i think its because of neoforge compatability

**Comment by dracoattila** [comment]
Will there be any chance the Connector will be updated/patched to operate well on Forge 47.2.0?

**Reply by Orionox** [reply]
same question

**Reply by FemBane** [reply]
I use 47.3.6, I'd love additional compatability
